### PR TITLE
(feat) Set background transparency levels.

### PR DESCRIPTION
**Pull request by horymury**
@saghul @quitrk @gimre @damencho @hristoterezov 

We can now set the background alpha through a config value.

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -646,6 +646,12 @@ var config = {
     */
     // dynamicBrandingUrl: '',
 
+    // Tileview toolbar button is shown and enabling tileview works even for a single participant.
+    // allowTileviewOnSingleParticipant: false,
```
Comment (saghul) [inline config.js:650]: Please keep this separate in another PR, as it's not related to the background alpha.

File: modules/API/API.js [hunk 2]
```diff
@@ -185,6 +185,9 @@ function initCommands() {
 
             APP.store.dispatch(toggleTileView());
         },
+        'set-tile-view': enabled => {
```
Comment (saghul) [inline modules/API/API.js:188]: Ditto, separate this in another PR.

File: react/features/base/util/helpers.js [hunk 3]
```diff
@@ -144,3 +144,30 @@ export function reportError(e: Object, msg: string = '') {
...
+export function setColorAlpha(color: string, opacity: number) {
+    if (!color) {
+        return `rgba(0, 0, 0, ${opacity})`;
+    }
+
+    let b, g, r;
+
+    if (color.startsWith('rgb')) {
+        [ r, g, b ] = color.split('(')[1].split(')')[0].split(',').map(c => c.trim());
+    } else if (color.startsWith('#')) {
```
Comment (saghul) [inline react/features/base/util/helpers.js:164]: What if the color is expressed with the shorthand `#fff` for example?

**Review by saghul — CHANGES_REQUESTED** [review]
Left some comments, PTAL!

File: react/features/base/util/helpers.js (hunk 3 above)
Comment (horymury) [reply react/features/base/util/helpers.js]: Added handler for shorthand colors, also guarding for any potential errors: send original color value on error parsing.

File: config.js [hunk 4]
```diff
@@ -673,6 +673,10 @@ var config = {
     */
     // dynamicBrandingUrl: '',
 
+
```
Comment (saghul) [inline config.js:676]: Drop this empty line. We generally leave 1 empty live between settings, and 2 between "sections".

**Review by saghul — APPROVED** [review]
LGTM with a nit, nice work!

**Review by saghul — APPROVED** [review]
